scripts/restore

```diff
@@ -94,12 +97,12 @@ chown root: $final_path/wp-config.php
 # RESTORE THE PHP-FPM CONFIGURATION
 #=================================================
 
-ynh_restore_file --origin_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
+ynh_add_fpm_config --usage=$fpm_usage --footprint=$fpm_footprint --package="$extra_php_dependencies"
```

Comment:
after rethinking of it... it'd be better if we use 
```
ynh_restore_file --origin_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
```
to restore the potentially modified php conf, and use `ynh_add_fpm_config` right after which will make a diff.
so:
```suggestion
# Restore the file first, so it can have a backup if different
ynh_restore_file --origin_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"

# Recreate a dedicated php-fpm config
ynh_add_fpm_config --usage=$fpm_usage --footprint=$fpm_footprint --package="$extra_php_dependencies"
```
(Sorry for my shitty suggestion)